doc/wiki/source_code/Cray.md

```diff
@@ -4,20 +4,34 @@ This page describes how to build mpich from the
 [main](https://github.com/pmodels/mpich/tree/main)
 branch of the MPICH git [repository](https://github.com/pmodels/mpich).
 
+## General
+
+Building on the Cray is like other systems: you'll need fairly
+up-to-date Autotools (autoconf \>= 2.67, automake \>= 1.15, libtool \>=
+2.4.4). On Argonne's theta machine, you can find these in
+`~robl/soft/cray/autotools`.  On Polaris, these seem to be new enough already.
+
+## Slingshot
+
+On Cray systems the vendor-supplied libfabric should be used to get slingshot support
+
+
+### Configure options
+
+You can leave off the 'cuda' options if you are e.g. testing only I/O or building.
+The Cray environemnt doesn't provide convienence environment variables for the 'pmi' or 'libfabric' paths, so you'll have to update these by hand when necessary
```

Comment:
You can use `CRAY_PMI_PREFIX` in place of the hardcoded path. Nothing I can see for libfabric, though.

Reply:
thanks, pushed new version with that fix